query-engine/js-connectors/js/smoke-test-js/prisma/sqlite-libsql/commands/type_test/insert.sql

```diff
@@ -0,0 +1,35 @@
+INSERT INTO type_test (
+    -- smallint_column,
+    int_column,
+    bigint_column,
+    -- float_column,
+    double_column,
+    -- decimal_column,
+    boolean_column,
+    -- char_column,
+    -- varchar_column,
+    text_column,
+    -- date_column,
+    -- time_column,
+    datetime_column
+    -- timestamp_column,
+    -- json_column,
+    -- enum_column
+) VALUES (
+    -- 32767, -- smallint
+    2147483647, -- int
+    9007199254740991 -- 9223372036854775807, -- bigint
+    -- 3.402823466, -- float
+    1.7976931348623157, -- double
+    -- 99999999.99, -- decimal
+    TRUE, -- boolean
+    -- 'c', -- char
+    -- 'Sample varchar', -- varchar
+    'This is a long text...', -- text
+    -- '2023-07-24', -- date
+    -- '23:59:59', -- time
+    '2023-07-24 23:59:59' -- datetime
+    -- '2023-07-24 23:59:59', -- timestamp
+    -- '{"key": "value"}', -- json
+    -- 'value3' -- enum
```

Comment:
Multiple of these values here make our `resultToColumnType` logic unhappy, so I manually edited values to make it work for now:
```
CREATE TABLE "type_test" (
    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
    "int_column" INTEGER NOT NULL,
    "int_column_null" INTEGER,
    "bigint_column" BIGINT NOT NULL,
    "bigint_column_null" BIGINT,
    "double_column" REAL NOT NULL,
    "double_column_null" REAL,
    "boolean_column" BOOLEAN NOT NULL,
    "boolean_column_null" BOOLEAN,
    "text_column" TEXT NOT NULL,
    "text_column_null" TEXT,
    "datetime_column" DATETIME NOT NULL,
    "datetime_column_null" DATETIME
);
INSERT INTO type_test VALUES (1, 2147483647, NULL, 2147483647, NULL, 1, NULL, 1, NULL, 'This is a long text...', NULL, '2023-07-24 23:59:59', NULL);
```